syntax = "proto3";

import "trade/vo/futures/FuturesProductInfoVo.proto";
option
java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response.futures";
option java_outer_classname = "FuturesQueryProductInfoResponseProto";
message FuturesQueryProductInfoResponse {
	repeated FuturesProductInfoVo productInfoVos = 1;
}
